crates/l2/prover/zkvm/interface/risc0/src/main.rs

```diff
@@ -31,12 +31,12 @@ fn main() {
     let receipts = execute_block(&block, &mut state).expect("failed to execute block");
     validate_gas_used(&receipts, &block.header).expect("invalid gas used");
 
-    env::write(
-        &receipts
-            .last()
-            .expect("no receipts found")
-            .cumulative_gas_used,
-    );
+    let cumulative_gas_used = match receipts.last() {
+        Some(last_receipt) => last_receipt.cumulative_gas_used,
+        None => 0_u64,
+    };
```

Comment:
```suggestion
    let cumulative_gas_used = receipts.last().map(|last_receipt| last_receipt.cumulative_gas_used).unwrap_or_default();
```

Reply:
Done in [c7aa3b7](https://github.com/lambdaclass/ethrex/pull/1466/commits/c7aa3b7938549869411b412f53caf64eac0bbb2e)